.vscode/settings.json

```diff
@@ -34,5 +34,10 @@
     "python.linting.pydocstyleArgs": [
         "--convention=google"
     ],
-    "python.linting.banditEnabled": true
+    "python.linting.banditEnabled": true,
+    "cpplint.lineLength": 120,
```

Comment:
Is there documentation for the `cpplint` settings? Does this apply to a particular extension?